diff -urN nethogs-0.3.org/nethogs.cpp nethogs-0.3/nethogs.cpp
--- nethogs-0.3.org/nethogs.cpp	2004-01-08 00:16:53.635221014 +0100
+++ nethogs-0.3/nethogs.cpp	2004-01-08 00:27:25.015599409 +0100
@@ -57,7 +57,7 @@
 	exit(0);
 }
 
-void main (int argc, char** argv)
+int main (int argc, char** argv)
 {
 #if DEBUG
 #else
@@ -107,5 +107,6 @@
 			needrefresh = false;
 		}
 	}
+	exit(0);
 }
 
diff -urN nethogs-0.3.org/process.cpp nethogs-0.3/process.cpp
--- nethogs-0.3.org/process.cpp	2004-01-08 00:16:53.654217089 +0100
+++ nethogs-0.3/process.cpp	2004-01-08 00:25:45.940120126 +0100
@@ -301,7 +301,7 @@
 	if (!DEBUG)
 	{
 		clear();
-		mvprintw (0, 0, caption->c_str());
+		mvprintw (0, 0, (char *)caption->c_str());
 		attron(A_REVERSE);
 		mvprintw (2, 0, "  PID USER     PROGRAM                                   Kbytes/s             ");
 		attroff(A_REVERSE);
